### PR TITLE
Enhancements to the ISmsClient

### PR DESCRIPTION
**Pull request by Cereal-Killa**
In reference to #222 
Simplifies the way an sms can be sent.

### CODE REVIEW

File: Vonage/Messaging/SmsClient.cs [hunk 1]
```diff
@@ -36,6 +36,16 @@ public SendSmsResponse SendAnSms(SendSmsRequest request, Credentials creds = nul
             return SendAnSmsAsync(request, creds).GetAwaiter().GetResult();
         }
 
+        public Task<SendSmsResponse> SendAnSmsAsync(string from, string to, string text, SmsType type = SmsType.text, Credentials creds = null)
+        {
+            return SendAnSmsAsync(new Messaging.SendSmsRequest { From = from, To = to, Type = type, Text = text }, creds);
```
Comment (slorello89) [inline Vonage/Messaging/SmsClient.cs:41]: Can you chop the `Messaging` scoping out of this? It should already be in the namespace :)

File: Vonage/Messaging/SmsClient.cs [hunk 2]
```diff
@@ -36,6 +36,16 @@ public SendSmsResponse SendAnSms(SendSmsRequest request, Credentials creds = nul
...
+        }
+
+        public SendSmsResponse SendAnSms(string from, string to, string text, SmsType type = SmsType.text, Credentials creds = null)
+        {
+            return SendAnSms(new Messaging.SendSmsRequest { From = from, To = to, Type = type, Text = text }, creds);
```
Comment (slorello89) [inline Vonage/Messaging/SmsClient.cs:46]: Same with this spot - just replace `Messaging.SendSmsRequest` with `SendSmsRequest`

**Review by slorello89 — CHANGES_REQUESTED** [review]
Looks great on my end - tested it out, makes much simpler thanks so much for this @Cereal-Killa. Just 2 ultra-minor items surrounding a namespace scoping.

**Comment by Cereal-Killa** [comment]
> Looks great on my end - tested it out, makes much simpler thanks so much for this @Cereal-Killa. Just 2 ultra-minor items surrounding a namespace scoping.

Absolutely, I missed that. It's all set now.

**Review by slorello89 — APPROVED** [review]
Looks great @Cereal-Killa - will push this out in the next release

**Pull request merged**